example-monorepos/blank/apps/next/next.config.js

```diff
@@ -13,12 +7,25 @@ const withTM = require('next-transpile-modules')([
   'dripsy',
   '@dripsy/core',
   'moti',
-  '@motify/core',
```

Comment:
Hi, should  `'@motify/core', '@motify/components',` be removed from the transpiler? I'm receiving this error when I do remove them.

```
error - node_modules\react-native-reanimated\lib\index.js:2
import './reanimated2/js-reanimated/global';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1032:15)
    at Module._compile (node:internal/modules/cjs/loader:1067:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (node_modules\@motify\core\lib\commonjs\use-map-animate-to-style.js:12:30)
    at Module._compile (node:internal/modules/cjs/loader:1103:14) {
  page: '/'
}
```

Reply:
Hey, yeah remove these from `next-transpile-modules` and then upgrade your moti version:

`yarn add moti@canary`